cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHttpService.java

```diff
@@ -63,6 +63,7 @@ public StreamHttpService(CConfiguration cConf, DiscoveryService discoveryService
       .setHandlerHooks(ImmutableList.of(new MetricsReporterHook(metricsCollectionService,
                                                                 Constants.Stream.STREAM_HANDLER)))
       .setHost(cConf.get(Constants.Stream.ADDRESS))
+      .setPort(cConf.getInt(Constants.Stream.PORT))
```

Comment:
In fact, AppFabric and DatasetService are the only services that run in the master itself, all others run in containers. It may still make sense to configure these ports for the other services for a standalone CDAP.